conformance/data_model/null.ion

```diff
@@ -0,0 +1,89 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+(ion_1_x "null values in text"
+         (then (text "null"          ) (denotes (Null          )))
+         (then (text "null.null"     ) (denotes (Null          )))
+         (then (text "null.blob"     ) (denotes (Null blob     )))
+         (then (text "null.bool"     ) (denotes (Null bool     )))
+         (then (text "null.clob"     ) (denotes (Null clob     )))
+         (then (text "null.decimal"  ) (denotes (Null decimal  )))
+         (then (text "null.float"    ) (denotes (Null float    )))
+         (then (text "null.int"      ) (denotes (Null int      )))
+         (then (text "null.list"     ) (denotes (Null list     )))
+         (then (text "null.sexp"     ) (denotes (Null sexp     )))
+         (then (text "null.string"   ) (denotes (Null string   )))
+         (then (text "null.struct"   ) (denotes (Null struct   )))
+         (then (text "null.symbol"   ) (denotes (Null symbol   )))
+         (then (text "null.timestamp") (denotes (Null timestamp))))
+
+(ion_1_x "malformed nulls in text should signal an error"
+         (each (text "null.")
+               (text "null.1")
+               (text "null.ion")
+               (text "null.array")
+               (signals "invalid type")))
+
+(ion_1_x "in text, unquoted null cannot be used as an annotation"
+         (text "null::0")
+         (signals "cannot use unquoted keyword as an annotation"))
```

Comment:
This is covered in a test case in `annotations.ion`.

Reply:
Oops. Guess I forgot to remove it here when I generalized it in `annotations.ion`